af04f855-58c4-4db3-a191-45fe33381679/scripts/actions.py

```diff
@@ -732,3 +749,37 @@ def moveOneRandom(group):
 	card.moveTo(me.hand)
 	notify("{} randomly moves {} from their discard to their hand.".format(me, card.name))
 	
+#------------------------------------------------------------------------------
+# Check Deck Tournament Validity
+#------------------------------------------------------------------------------
+
+LegacyLegal=["01-Core Set", "02-Warlord Cycle", "03-The Great Devourer", "04-Planetfall Cycle" "05-Legions of Death", "06-Deathworld Cycle"]
```

Comment:
It is up to each tournament to define what is legal for the tournament. There could be a warlord cycle tournament or an all FFG cards with champion cards tournament, or any variety. Every one of those combinations is not going to appear in this selection box, and it doesn't help with enforcement since an illegal card can never be played without showing that the deck is illegal.

Where built in deck validation helps is with things like number of cards, because it can benefit a player through increasing chance of drawing a card while not revealing that the deck is illegal (holding excess copies in their hand).

The only way I see this being used is for someone to check for themselves if they've made a legal deck, but again since it's on a per tournament basis we would have to support every possible option for it to be generally useful.

Further, I don't like how it pushes the agenda item of whether or not champion cards are OK for play. By adding this option, I see people using it as a way to say for casual play, that champions card are not OK to be used, which is not the case. That's up to two individuals playing cards, just like if they sat down at a table.

With multiple leagues I see some value in being able to say is this FFG, Apoka, or BCL, but it all puts the champion cards in a weird spot and I'm more inclined to not have that option and just have

Verify deck format
Apoka or BCL (which isn't needed at the moment since there's been no demand)

and not try to define what FFG is

Reply:
The option clearly state that it is made for Tournaments, and is not enforced in any way in the original deck checker at the load of the deck (which I could do) in order to prevent the specific case you state. (that some cards are not okay in any way). It's mostly to support the current tournaments (so not casual play) that takes place despite the end of the game. So I took on me to choose thoses 2 options as still relevant (again, at the moment), and will evolve them later if there were a need (or people who ask for them).

The champions cards are already in a weird spot (see the whole blocs of texts on facebook groups). And I can't really take any decision concerning them myself (as I play only casually). I won't be able to turn a blind eye about them either as I took on me to maintain the game for years. 
It will mostly be to the current tournament scene to consider them and to decide to add them (which they don't at the moment). As casual cards they are fine as it is. As a tournament cards, they will probably never be allowed again (they're not available in any way apart from digitally, only a "niche group" had access to it, and even then, there's no good scans of them, they are still considered promo cards by all current players. Also no official statement about them in any way by Brad Antes, we just know that he printed them as a gift for Sam and the champions of the game).

So there are my choices and opinions : 
- Supporting FFG + Apoka as it is for now. Adding Champions Cards or/and BCL if needed (as I follow regularly all channels carddb, facebook, and discord)
- Supporting Apoka only. And I'm not okay with that as it would remove some neutral view about the current scene (some countries still play the official way, it's not the majority at all on octgn, but if it were to happen, I prefer to have the option ready. If I found current "ffg" tournament, I didn't found any of them supporting champions cards, and/or BCL, I really did my research here).
- Supporting FFG, Apoka, FFG + Champions Cards, BCL (which, I think, doesn't use the champions cards if I remember well from my discussion with sam, but I'll ask again if it were to be the option you reaaaally want) and 2 of thoses options will probably never be used (against because the current scene does not use them), and clutter the menu. Which I'm also not really okay with.

As long as the option clearly states that it is for tournament play, I would prefer it to stay that way.